src/modules/expense/Form.tsx

```diff
@@ -0,0 +1,226 @@
+        <Select
+          labelId='select-category'
+          id='simple-select-category'
+          multiple
+          value={category}
+          label='Category'
+          onChange={handleChangeCategory}
+          input={<OutlinedInput id='select-multiple-chip' label='Category' />}
+          renderValue={(selected) => (
+            <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: 0.5 }}>
+              {selected.map((value) => (
+                <Chip
+                  key={value}
+                  label={value}
+                  sx={{
+                    backgroundColor: colors.light[60],
+                    border: '1px solid ' + colors.light[80],
+                  }}
+                />
+              ))}
+            </Box>
+          )}
+          MenuProps={MenuProps}
+        >
+          {categories.map((category) => (
+            <MenuItem key={category} value={category}>
+              {category}
+            </MenuItem>
+          ))}
+        </Select>
+        <TextField
+          label='Description'
+          value={textValue}
+          onChange={handleChangeText}
+          variant='outlined'
+          sx={{ marginTop: '20px' }}
+        />
+        <FormControl sx={{ marginTop: '20px' }}>
+          <InputLabel id='select-option'>Wallet</InputLabel>
+          <Select
+            labelId='select-option'
+            id='simple-select-option'
+            value={selectedWallet}
+            onChange={handleChangeOption}
+            input={<OutlinedInput id='select-single-option' label='Option' />}
+          >
+            <MenuItem value='Paypal'>Paypal</MenuItem>
+            <MenuItem value='Google Pay'>Google Pay</MenuItem>
+            <MenuItem value='Apple Pay'>Apple Pay</MenuItem>
+          </Select>
+          <Button
+            onClick={() => setShowAttachmentComponent(!showAttachmentComponent)}
+            component='label'
+            sx={{
+              backgroundColor: 'white',
+              border: '1px dashed gray',
+              height: '56px',
+              marginTop: '20px',
+              color: colors.dark[50],
+            }}
+          >
+            <AttachFileIcon sx={{ marginRight: '10px' }}></AttachFileIcon>
+            Add attachment
+            {/* <input type='' hidden /> */}
+          </Button>
+        </FormControl>
+        {showContinueButton && <Button>Continue</Button>}
+      </FormControl>
+      {showAttachmentComponent && (
+        <StyledDivAttachmentArea
+          style={{
+            width: '100%',
+            height: '171px',
+            backgroundColor: colors.light[60],
+            borderRadius: '24px 24px 0 0',
+          }}
+        >
+          <Stack
+            direction='row'
+            spacing={2}
+            padding={3}
+            sx={{
+              width: '100%',
+              display: 'flex',
+              justifyContent: 'space-evenly',
+              marginTop: '30px',
+            }}
+          >
+            <Button
```

Comment:
Create a shared component for each of these buttons so you don't need to repeat yourself